kernel/src/syscall/pread64.rs

```diff
@@ -31,12 +27,15 @@ pub fn sys_pread64(
     }
 
     let read_len = {
+        let filetable = ctx.process.file_table().lock();
         let mut writer = ctx
             .process
             .root_vmar()
             .vm_space()
             .writer(user_buf_ptr, user_buf_len)?;
-        file.read_at(offset as usize, &mut writer)?
+        filetable
+            .get_file(fd)?
+            .read_at(offset as usize, &mut writer)?
```

Comment:
This will cause blocking while holding the `file_table` lock. I'm afraid this is undesirable.

Reply:
Ok, I will discard this commit since it had minimal impact on the performance improvement.